pkg/pipeline/decode/decode_protobuf.go

```diff
@@ -0,0 +1,85 @@
+package decode
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/netobserv/flowlogs-pipeline/pkg/config"
+	"github.com/netobserv/netobserv-agent/pkg/pbflow"
+	"github.com/sirupsen/logrus"
+)
+
+var pflog = logrus.WithField("component", "Protobuf")
+
+// Protobuf decodes protobuf flow records definitions, as forwarded by
+// ingest.NetObservAgent, into a Generic Map that follows the same naming conventions
+// as the IPFIX flows from ingest.IngestCollector
+type Protobuf struct {
+}
+
+func NewProtobuf() (Decoder, error) {
+	return &Protobuf{}, nil
+}
+
+// Decode decodes input strings to a list of flow entries
+func (c *Protobuf) Decode(in []interface{}) []config.GenericMap {
```

Comment:
Are you using "c *Protobuf" in the function ???

Reply:
No, it is not used, but the API conventions of the flowlog pipeline require that `Decode` is run as a method of a type implementing the `Decoder` interface.

Reply:
copy/paste issues. Changing to `p` or even `_`, as it is unused.